h/views/api_auth.py

```diff
@@ -49,26 +87,15 @@ def get(self):
 
         state = credentials.get('state')
         user = self.user_svc.fetch(self.request.authenticated_userid)
+        response_mode = credentials.get('request').response_mode
```

Comment:
Yes, this is an `oauthlib` request. I've tried to be clear about the naming to not cause confusion. This is, unfortunately, the return value of an oauthlib call, which uses `request`.